xbmc/GUIInfoManager.cpp

```diff
@@ -2118,6 +2118,15 @@ const infomap videoplayer[] =    {{ "title",            VIDEOPLAYER_TITLE },
 /// @{
 /// \table_start
 ///   \table_h3{ Labels, Type, Description }
+///   \table_row3{   <b>`VideoFilter`</b>,
```

Comment:
There is something wrong with those infolabels. I created a simple service addon to poll them every second because the way they are documented is confusing. Normally you document them along with their namespace: `RetroPlayer.VideoFilter` instead of just `VideoFilter`. I tried several combinations while 2048 was opened and the only infolabel I was able to get was the strechmode one. Service is below:

```python
def run():
    monitor = xbmc.Monitor()

    while not monitor.abortRequested():
        # Sleep/wait for abort for 10 seconds
        if monitor.waitForAbort(1):
            # Abort was requested while waiting. We should exit
            break
        print("Get RetroPlayer Infobools")
        print("stretchmode: %s" % xbmc.getInfoLabel('retroplayer.stretchmode'))
        print("Videofilter: %s" % xbmc.getInfoLabel('retroplayer.videofilter'))
        print("VideoRotation: %s" % xbmc.getInfoLabel('retroplayer.videorotation'))
```

Kodi log shows the following:
```
│10:27:38.287 T:123145554681856   DEBUG: Get RetroPlayer Infobools                                  │
│10:27:38.287 T:123145554681856   DEBUG: stretchmode: 4:3                                           │
│10:27:38.287 T:123145554681856   DEBUG: Videofilter: retroplayer.videofilter                       │
│10:27:38.288 T:123145554681856   DEBUG: VideoRotation: retroplayer.videorotation
```
Which indicates both `retroplayer.videofilter` and `retroplayer.videorotation` are not being recognized. Can you check what is happening and please change those entries in the documentation to their "full name"?